graphql_api/types/plan/plan.graphql

```diff
@@ -6,6 +6,7 @@ type Plan {
   pretrialUsersCount: Int
   marketingName: String!
   planName: String!
+  tierName: String!
```

Comment:
Do we use the GraphQL enum anywhere?  Should this be `tierName: TierName!` instead?